src/cascadia/TerminalSettingsModel/Microsoft.Terminal.Settings.ModelLib.vcxproj

```diff
@@ -10,7 +10,12 @@
     <SubSystem>Console</SubSystem>
     <OpenConsoleUniversalApp>true</OpenConsoleUniversalApp>
   </PropertyGroup>
+  <PropertyGroup Label="NuGet Dependencies">

```

Comment:
In some projects, you put these in their own property group and in others you didn't -- should we standardize on one?

(The answer to this precludes a different question I had, which was "Should we rename the per-package properties to make it clear that they're about packages?" If they're in this group, perhaps they can stay as-is)

Reply:
It should probably be in its own group but I didn't go back to reconcile them all (I did cppwinrt first and many projects only needed that).  I can make it consistent if it is desirable.  

The names for opting into dependencies could be modified to be clearer if you have a suggested scheme.  I think there is some danger of colliding with the "real" names of properties so I used "Terminal" as a prefix on them all to prevent that.

Reply:
This vcxproj was the only project where I missed an include.  I fixed it.  Good catch!